src/plugins/kv-analytics-plugin.js

```
@@ -55,6 +58,17 @@ export default Vue => {
 				window.ga('send', 'pageview');
 			}
 
+			if (gaAltLoaded) {
+				let gaAltId;
+				/* eslint-disable no-underscore-dangle */
+				if (window.__KV_CONFIG__ && window.__KV_CONFIG__.gaAlternateId) {
+					gaAltId = window.__KV_CONFIG__.gaAlternateId;
+				}
+				window._gaq.push(['_setAccount', gaAltId]);
+				window._gaq.push(['_trackPageview']);
```

If that config wasn't available these would fail silently.